packages/react/src/components/index.ts

```diff
@@ -9,6 +9,7 @@ export * from './dialog';
 export * from './dropdown';
 export * from './errorSummary';
 export * from './fieldset';
+export * from './fileInput';
```

Comment:
Not sure if we want to export the _ formatBytes_ function also from the file? Though not sure if it matters really

Reply:
I only exported it because of the tests. I'm not sure if it has any usage outside the component.